ecosystem/README.md

```diff
@@ -1,171 +1,98 @@
-| [SEP-0014](sep-0014.md) | Dynamic Asset Metadata | OrbitLens, Paul Tiplady | Standard | Draft |
-| [SEP-0015](sep-0015.md) | Attachment Convention | Interstellar | Standard | Draft |
-| [SEP-0016](sep-0016.md) | Account Transfer Permissionless Payment Protocol (@p2p) | Jeremy Rubin | Standard | Draft |
-| [SEP-0017](sep-0017.md) | Issuer account funding protocol (CAP-13 Based) | Tom Quisel | Standard | Draft |
-| [SEP-0018](sep-0018.md) | Data Entry Namespaces | Mister.Ticot | Standard | Draft |
-| [SEP-0019](sep-0019.md) | Bootstrapping Multisig Transaction Submission | Paul Selden, Nikhil Saraf | Standard | Draft |
-| [SEP-0021](sep-0021.md) | On-chain signature & transaction sharing | Mister.Ticot | Informational | Draft |
-| [SEP-0022](sep-0022.md) | IPFS Support | Samuel B. Sendelbach | Informational | Draft |
-| [SEP-0023](sep-0023.md) | Augmented strkey format for multiplexed addresses | David Mazières and Tomer Weller | Standard | Draft |
-| [SEP-0029](sep-0029.md) | Account Memo Requirements | OrbitLens, Tomer Weller, Leigh McCulloch, David Mazières | Standard | Draft |
-
-
-### Rejected and Deprecated Proposals
-
-| Number | Title | Author | Track | Status |
-| --- | --- | --- | --- | --- |
-| [SEP-0003](sep-0003.md) | Compliance Protocol | SDF | Standard | Deprecated |
-| [SEP-0006](sep-0006.md) | Anchor/Client Interoperability | SDF | Standard | Deprecated in favor of SEP-24 |
-| [SEP-0013](sep-0013.md) | DEPOSIT_SERVER proposal | @no, @ant, @manran, @pacngfar | Informational | Rejected |
-
-
-# Contribution Process
-
-The Stellar Ecosystem, like most software ecosystems in the world, continues to evolve over time to
-meet the needs of our network's participants and to drive technology forward into new territory.
-
-Unlike Stellar's Core development (CAPs), Stellar's Ecosystem Proposals are intended to be a more
-dynamic way of introducing standards and protocols utilized in the ecosystem that are built on top
-of the Stellar Network. It attempts to take a more lightweight process for approval, and much of
-its process is inspired by the [IETF][ietf].
-
-Before contributing, consider the following:
-
-- Choose a track to propose your idea on. The bar for accepting an `Informational` SEP is much
-  lower than one for a `Standard`, and allows you to promote the SEP independently to gain feedback
-  and traction before creating a Standard out of it.
-- Gather feedback from discussion on the dev mailing list and other forums, and utilize it to begin
-  a draft proposal.
-- Follow the proposal process listed below. If you're having difficulty moving the proposal
-  forward, talk to the buddy that's assigned the SEP; they'll often have guidance on how to move
-  things forward, as well as feedback regarding feasibility and how the proposal does or does not
-  align with the Stellar Network's goals.
+* **Draft** - A proposal that is currently in discussion and undergoing revision.  Drafts remain as pull requests until approved.
```

Comment:
I was really into this idea of keeping drafts in PRs when we talked about this a few weeks ago, but it occurred to me that having such a different process for SEPs compared to CAPs is confusing and likely to mean any process we document will not be something we follow consistently.

It's also really beneficial if we allow drafts to be iterated on in a central place that's accessible to everyone making contributions and while a PR makes that possible to a degree, it doesn't encourage it.

I like @stanford-scs's recent suggestion that we keep drafts fluid and quickly update-able. I think we absolutely can use PRs as the place to discuss a draft if the author chooses to by specifying a [Discussion URL](https://github.com/stellar/stellar-protocol/blob/072792efc9c0790742d0dc275a9ba63a39063603/sep-template.md#L10) like we already have in our templates. If we ensure we always use PRs for them, anyone can always comment on updates even if a PR has been merged, or if a PR merges too quickly they can even open a PR themselves proposing another direction for a SEP. Or an author can choose to have discussion in the mailing list or anywhere else of their choosing.

I think the issue that led, at least me, to think drafts should stay in PRs is that looking at the repo I've struggled to tell which SEPs/CAPs are drafts vs concrete things to rely on. But that's because for the longest time I didn't realize there was a README.md in the ecosystem directory (it is pushed so far down the page because the directory has so many files in it). #556 attempts to help others not make the same mistake I did.

We could make it clearer that a SEP is a draft by not giving it a number until it moves out of draft, this is something RFCs do. But that also has some downsides with git because renaming a file can make it harder to follow its history, so I don't think it's worth going that far.

cc @rice2000 @msfeldstein @tyvdh